tests/main.tf

```diff
@@ -36,7 +37,6 @@ module "primary-cluster-regional-nat" {
   zones                                = "${var.zones}"
   project                              = "${var.project}"
   environment                          = "${terraform.workspace}"
-  min_master_version                   = "${var.kube_version}"
```

Comment:
What has happened with `min_master_version`?

Reply:
test is failing when defined kube_version becomes obsolete